blueprints/addon/files/.travis.yml

```diff
@@ -45,8 +45,6 @@ before_install:<% if (yarn) { %>
 install:
   - yarn install --no-lockfile --non-interactive
 <% } else { %>
-  - npm config set spin false
-  - npm install -g npm@4
   - npm --version
```

Comment:
we don't need this line either, since it was printing the newly installed npm version

Reply:
Removed